src/services/database/create-message-fragment.js

```diff
@@ -0,0 +1,20 @@
+import { runWithinTransaction } from './helper';
+import { logError, logInfo } from "../../middleware/logging";
+import ModelFactory from '../../models';
+import { modelName, Status } from '../../models/message-fragment';
+
+const MessageFragment = ModelFactory.getByName(modelName);
+
+export const createMessageFragment = (messageId, conversationId) =>
+  runWithinTransaction(transaction =>
+    MessageFragment.create(
+      {
+        messageId,
+        conversationId,
+        status: Status.FRAGMENT_REQUEST_RECEIVED
+      },
+      transaction
+    )
+      .then(() => logInfo('Fragments trace has been stored'))
```

Comment:
```suggestion
      .then(() => logInfo('Fragments trace has been stored'))
```
No longer called 'Fragments trace'. Replace with 'Message fragment'.

Reply:
Worth trying to search for all references to 'fragments trace'.